### PR TITLE
Switch to sumo

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Switch to Poky version sumo-19.0.0
```
Comment (spanceac) [inline CHANGELOG.md]: You forgot to add your name here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Make sure resin-data.img is deployed [florin]
```
Comment (spanceac) [inline CHANGELOG.md]: You accidentally typed your name with the first letter lowercase

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by floion** [comment]
retest

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by floion** [comment]
@resin-ci retest

**Pull request merged**